### ISSUE TITLE
ValueError: invalid literal for int() with base 10

### ISSUE THREAD

**Issue by rizkiaditya24** [bkabrda/flask-whooshee#40]
I got an error when running flask-whooshee with primary key is a string, not an integer. Here is my model code:

```
@whooshee.register_model('short_text', 'status')
class User(db.Model):
    __tablename__ = 'users'

    id = db.Column(db.String, primary_key=True)
    short_text = db.Column(db.String)
    status = db.Column(db.String)
```

I've initialized the app using init_app in my create_app function.

How to solve this error?

**Comment by bkabrda** [comment]
Fixed and released as part of version 0.6.0: https://pypi.org/project/flask-whooshee/0.6.0/

**Comment by bkabrda** [comment]
(To be more precise, this should now just work with no changes on your end. Feel free to let me know if it works as expected.)
